liquidcore/home/templates/registration/login.html

```diff
@@ -57,8 +59,14 @@ <h1>Login</h1>
   <input type="hidden" name="next" value="{{ next }}" />
 </form>
 
-{% if not form.otp_token %}
-<p><a href="{% url 'password_reset' %}">Lost password?</a></p>
-{% endif %}
+<footer class="page-footer font-small blue pt-4">
+
+  {% if not form.otp_token %}
+  <p><a href="{% url 'password_reset' %}">Lost password?</a></p>
+  {% endif %}
+
+  <i>Note</i> All hyphens (<pre>-</pre>) in usernames have been replaced with dots (<pre>.</pre>).
```

Comment:
Ugh, let's not keep this around forever. It's just 3 users, much easier to tell them directly.

Reply:
Ok, I'll kill this.